xml/System.Net.NetworkInformation/PhysicalAddress.xml

```diff
@@ -355,7 +359,7 @@
       </Docs>
     </Member>
     <Member MemberName="Parse">
-      <MemberSignature Language="C#" Value="public static System.Net.NetworkInformation.PhysicalAddress Parse (string? address);" FrameworkAlternate="net-5.0;net-6.0" />
+      <MemberSignature Language="C#" Value="public static System.Net.NetworkInformation.PhysicalAddress Parse (string? address);" FrameworkAlternate="net-5.0;net-6.0"/>
```

Comment:
This is autogenerated so it will just be overwritten in the next update.

Reply:
Accidentally removed that space - added it back in for sake of diff.